admin/base/settings/defaults.py

```diff
@@ -25,18 +25,34 @@
     '.osf.io'
 ]
 
+# Email settings. Account created for testing. Password shouldn't be hardcoded
```

Comment:
Might be worth looking into this https://docs.djangoproject.com/en/dev/topics/email/#console-backend OSF uses a similar concept for development.